servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java

```diff
@@ -3526,8 +3526,19 @@ void validateInstanceServiceIdentity(DomainData domainData, final String service
         // need to check anything
 
         final String domainName = domainData.getName();
-        if (validateServiceSkipDomains.contains(domainName)) {
-            return;
+        if (domainName != null) {
```

Comment:
one more comment - looking at the server code, this can never be null so we should remove It, otherwise it's missing unit tests.

Reply:
Thank you. 
Since it has been determined that domainName will not be null, I have removed the branching.
Additionally, I've made modifications to the test code to specify the domainName.